tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/AbstractPortalBundle.java

```diff
@@ -404,4 +407,9 @@ else if( configType.equals( CONFIG_TYPE_SERVER ) )
             }
         }
     }
+
+    public IPath getLogPath()
+    {
+        return getLiferayHome().append( "logs" ).append( "Liferay." + sdf.format( new Date() ) + ".log" );
```

Comment:
This file is a extension for normal log4j.xml?  We need to analyze this content to find the log stored location?